cite_seq_count/preprocessing.py

```diff
@@ -57,7 +57,7 @@ def parse_barcode_reference(
     barcode_pattern = rf"^[ATGC]{{{barcode_length}}}"
 
     header = barcodes_pl.columns
-    set_dif = set(required_header) - set(header)
+    set_dif = set([required_header]) - set(header)
```

Comment:
without the brackets it was splitting every letter of the required_header

Reply:
Keep that one in, I'll merge once the PR is ready